src/functions/schema.ts

```diff
@@ -100,26 +105,28 @@ const validators = new (class extends WeakMap<JSONSchema, ValidateFunction> {
   }
 })();
 
-const replaceProperty = (substring: string, group1: string) => {
-  if (group1) {
-    return 'property ';
+const replaceProperty = (substring: string, _: Optional<number | string>, propertyName: Optional<string>) => {
+  if (typeof _ === 'string' && propertyName !== void 0) {
```

Comment:
@P0lip As it's actually used in the code, can't we give this `_` param a more meaningful name?